packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -105,7 +105,7 @@ export const start = async (zcf, privateArgs) => {
     }
   };
 
-  /** @type {Store<Brand,VaultManager>} */
+  /** @type {Store<Brand,import('./vaultManager').VaultManager>} */
```

Comment:
a little cumbersome to import but the graph of what types are needed where can help us to refactor for better modularization

Reply:
I really want to avoid this construct. That spreads interfile dependencies all over.

Reply:
> I really want to avoid this construct.

Agree it's ugly here. It's a lot cleaner in .ts syntax.

> That spreads interfile dependencies all over.

Counterpoint: there was an implicit dependency before and now it's explicit.

Reply:
Would a `@typedef {import()}` at the top be sufficient? They are the logical equivalent to TypeScript's `import type`. We have explicit imports for runtime identifiers, it doesn't shock me to have them explicit for types as well.